### PR TITLE
Added github action for jetbrains/qodana-jvm-community

### PR DESCRIPTION
**Pull request by ak1113**

### CODE REVIEW

File: run_qodana.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    $additional_volumes \
+    $additional_env_variables \
+    $qodana_image \
+    $(if [ -n "$INSPECTED_DIR" ]; then echo "-d $INSPECTED_DIR"; fi) \
+    $(if [ -n "$BASELINE_PATH" ]; then echo "-b $BASELINE_PATH"; fi) \
+    $(if [ "$BASELINE_INCLUDE_ABSENT" == "true" ]; then echo "--baseline-include-absent"; fi) \
+    $(if [ -n "$FAIL_THRESHOLD" ]; then echo "--fail-threshold $FAIL_THRESHOLD"; fi) \
+    $(if [ "$SAVE_HTML_REPORT" == "true" ]; then echo "--save-report"; fi) \
+    $(if [ -n "$PROFILE_NAME" ]; then echo "-profileName $PROFILE_NAME"; fi) \
+    $(if [ -n "$PROFILE_PATH" ]; then echo "-profilePath $FAIL_THRESHOLD"; fi)
```
Comment (CynicalFrog) [inline run_qodana.sh]: $(if [ -n "$PROFILE_PATH" ]; then echo "-profilePath $PROFILE_PATH"; fi) ?

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+```
+
+All action's inputs are optional. 
+```yaml
+- name: Qodana - JVM
+  uses: JetBrains/qodana-jvm-community-action@v1.1.1
+  with:
+      fail-threshold: 10
+      additional-env-variables: |
+            IDE_PROPERTIES_PROPERTY='-Didea.headless.enable.statistics=false;-Didea.required.plugins.id=JavaScript,org.intellij.grails'
```
Comment (CynicalFrog) [inline README.md]: Is this code going to be public? We have QD-1280 regarding this variable, but I'm not sure how to get around it yet.

File: run_qodana.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+qodana_image='jetbrains/qodana-jvm-community:2021.3' #todo: verfiy version
```
Comment (CynicalFrog) [inline run_qodana.sh]: This is going to be jetbrains/qodana-jvm-community:latest or just jetbrains/qodana-jvm-community for now. The build is ready, we just need to run tests and get approval from everyone.

File: README.md (hunk 2 above)
Comment (ak1113) [reply README.md]: I was not aware of the QD-1280 issue. Removed the example

File: run_qodana.sh (hunk 3 above)
Comment (ak1113) [reply run_qodana.sh]: Changed to `jetbrains/qodana-jvm-community:2021.2`

**Pull request merged**